.github/workflows/build.yml

```diff
@@ -87,10 +87,6 @@ jobs:
       run: |
         docker-compose -f ./sdk-tests/deploy/local-test-kafka.yml up -d
         docker ps
-    - name: Install Local Hashicorp Vault using docker-compose
```

Comment:
have the vault install in the build.yaml and remove it from the validate.yaml file ....

Reply:
ok, and the Integration Test was fail because the vault secret test case. should i fix it ?

Reply:
Yes please @dntbeSris 
cc @artursouza

Reply:
done